lib/pipeline.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Membrane.VideoCompositor.Pipeline do
+  @moduledoc """
+  Pipeline for testing simple composing of two videos, by placing one above the other.
+  """
+
+  use Membrane.Pipeline
+
+  @doc """
+  handle_init(%{
+      paths: %{
+        first_raw_video_path: String.t(),
+        second_raw_video_path: String.t(),
+        output_path: String.t()
+      },
+      caps: RawVideo,
+      implementation: :ffmpeg | :opengl | :nx
+  })
+  """
+  @impl true
+  def handle_init(options) do
+    children = %{
+      first_file: %Membrane.File.Source{location: options.paths.first_raw_video_path},
+      second_file: %Membrane.File.Source{location: options.paths.second_raw_video_path},
+      first_parser: %Membrane.RawVideo.Parser{
+        framerate: options.caps.framerate,
+        width: options.caps.width,
+        height: options.caps.height,
+        pixel_format: options.caps.pixel_format
+      },
+      second_parser: %Membrane.RawVideo.Parser{
```

Comment:
```suggestion
      file_src_1: %Membrane.File.Source{location: options.paths.first_raw_video_path},
      file_src_2: %Membrane.File.Source{location: options.paths.second_raw_video_path},
      parser_1: %Membrane.RawVideo.Parser{
        framerate: options.caps.framerate,
        width: options.caps.width,
        height: options.caps.height,
        pixel_format: options.caps.pixel_format
      },
      parser_2: %Membrane.RawVideo.Parser{
```
I think names like that are more readable